v2/stories/ArticleMetadata.stories.js

```diff
@@ -0,0 +1,36 @@
+import VByline from '../src/components/VByline.vue'
+import VCounter from '../src/components/VCounter.vue'
+
+export default {
+    title: 'Components-V2/ArticleMetadata',
+    components: {
+        VByline,
+        VCounter
+    }
+}
+
+const Template = ( args ) => ( {
+    components: {
+        VByline,
+        VCounter
+    },
+    setup() {
+        return {
+            args
+        }
+    },
+    template: '<div class="article-metadata"><v-byline v-bind="args" /><div class="article-metadata-separator" /><v-counter text="Comments" :value="40" href="http://www.google.com" /></div>'
```

Comment:
looks like you don't need the "<div class="article-metadata-separator" />" here... looks like css is already in place to handle the separator.